src/sections/Info.jsx

```diff
@@ -84,7 +84,18 @@ const PushinP = styled.p`
   }
 `
 
+const fishingBearBob = keyframes`
+  0%, 100% {
+    top: -2vw; 
```

Comment:
have you tried having tranform for your keyframes instead of absolute positioning value? maybe it'll make things look a bit smoother

```
0%, 100% {
  transform: translate(0, -2vw);
}
```

Reply:
ty for suggestion, that fixed it ^